build.gradle

```diff
@@ -53,6 +54,15 @@ allprojects {
     }
 }
 
+apply plugin: 'com.github.leandroborgesferreira.dag-command'
+
+dagCommand {
+    filter = "all"
+    defaultBranch = "origin/develop"
```

Comment:
What happens on PRs where we're not targeting this branch? It's also odd that it includes `origin`, which doesn't make sense in the CI context (assuming that's the eventual goal, applying this there). People might have different names for the git remote locally as well.

Reply:
The PRs that don't target `origin/develop`, could have the normal unit test and Ktlint flow, the idea is to speed up only the merge with `develop` because this is where speed has value (ex: merge with `main` could take longer). About the remote, let's change it to just `develop`, so it doesn't become a problem with different remotes.